src/coreclr/src/jit/gentree.h

```diff
@@ -2853,6 +2861,19 @@ struct GenTreeOp : public GenTreeUnOp
         assert(oper == GT_NOP || oper == GT_RETURN || oper == GT_RETFILT || OperIsBlk(oper));
     }
 
+    // returns true if we will use MagicNumber multiplication for this node.
+    bool usesMagicNumberDivision(Compiler* comp);
```

Comment:
Nit: we prefer to start function with an upper case unless we create them with a prefix or in an old class that does not follow the current cc, so `UsesMagicNumberDivision/CheckMagicNumberDivision` etc.

Reply:
Done as part of #39021